Makefile

```diff
@@ -1,18 +1,29 @@
 RUSTUP = rustup
+
 TOOLCHAIN = $(shell cat rust-toolchain)
-CARGO = $(RUSTUP) run --install $(TOOLCHAIN) cargo --color always
+CARGO = $(RUSTUP) run $(TOOLCHAIN) --install cargo --color always
+
+NIGHTLY_TOOLCHAIN = "nightly-2019-07-31"
+CARGO_NIGHTLY = $(RUSTUP) run $(NIGHTLY_TOOLCHAIN) --install cargo --color always
 
 build: build_debug
 
+install_rust:
```

Comment:
📣  This is not needed, `--install` does it for you.

Reply:
https://circleci.com/gh/comit-network/create-comit-app/711 :(

Reply:
`--install` only works for the subcommand `run`. It doesn't work for the sub command `components` which we execute first in order to see if we have all the components installed ...

Reply:
dang!